### ISSUE TITLE
Links in SEE ALSO section of doc/lore/man/lore.1 are bogus

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1971]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1971|
|Type|defect|
|Created|2006-07-29 20:19:01Z|
|Branch|https://github.com/twisted/twisted/tree/bogus-links-1971|

<details><summary>Searchable metadata</summary>

```
trac-id__1971 1971
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_bogus_links_1971 branches/bogus-links-1971
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation documentation
time__1154204341000000 1154204341000000
changetime__1217871776000000 1217871776000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

Which file is that exactly?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

`doc/lore/man/lore.1`

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [24456]) Branching to 'bogus-links-1971'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

This [page](http://www.schweikhardt.net/man_page_howto.html#q3) writes that 'the SEE ALSO section is a list of related man pages in alphabetical order'. The filenames currently listed in the page are not man pages but refer to .xhtml documents. I suggest we remove the SEE ALSO section to avoid confusion. Updated the branch, putting it up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24515]) Merge bogus-links-1971: removing pointless links from lore man page.

Author: thijs
Reviewer: exarkun
Fixes: [#1971](https://github.com/twisted/twisted/issues/1971)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
